.buildkite/pipeline.ml.yml

```diff
@@ -5,6 +5,7 @@
     - DATA_PROCESSING_TESTING=1 INSTALL_HOROVOD=1 ./ci/env/install-dependencies.sh
     - bazel test --config=ci $(./ci/run/bazel_export_options) --build_tests_only --test_tag_filters=-gpu,-needs_credentials python/ray/air/...
     - bazel test --config=ci $(./ci/run/bazel_export_options) --build_tests_only --test_tag_filters=ray_air,-gpu_only,-gpu,-needs_credentials python/ray/train/...
+    - bazel test --config=ci $(./ci/run/bazel_export_options) --build_tests_only --test_tag_filters=ray_air python/ray/data/...
```

Comment:
Just leaving a note, but in a future PR we should split up the AIR tests into multiple test suites so they can run in parallel.

Reply:
Yeah, definitely. I have some ideas on how to do that, but right now, this isn't the bottleneck in CI. Regardless, it's orthogonal to this